pkg/clusters/addons/metallb/metallb.go

```diff
@@ -144,7 +144,7 @@ func (a *Addon) DumpDiagnostics(context.Context, clusters.Cluster) (map[string][
 // -----------------------------------------------------------------------------
 
 var (
-	metalManifest = "https://github.com/metallb/metallb/config/native?ref=v0.13.12&timeout=2m"
+	metalManifest = "https://github.com/metallb/metallb/config/native?ref=v0.14.8&timeout=2m"
```

Comment:
Would it be worth covering with renovate? But does it need to be in sync with the version from `team-k8s` K8s scripts? Just food for thought

Reply:
Renovate would be nice to add here 👍 

Let's handle that in a separate PR. Created https://github.com/Kong/kubernetes-testing-framework/issues/1163 to track this.